src/histolab/filters/compositions.py

```diff
@@ -47,6 +55,49 @@ def __new__(cls: type, cls_):
             f"Filters composition for the class {cls_.__name__} is not available"
         )
 
+    @lazyproperty
+    def tissue_mask_filters(self) -> imf.Compose:
```

Comment:
why we need this here? I don't see the reason of the empty method in here only for the doc. Maybe we should use a `raise NotImplementedError` in the super class otherwise it looks weird to me